client/package.json

```diff
@@ -7,11 +7,15 @@
     "test": "jest"
   },
   "dependencies": {
+    "apollo-client": "^2.6.2",
+    "graphql": "^14.3.1",
     "graphql-tag": "^2.10.1",
     "react": "16.8.3",
     "react-apollo": "^2.5.6",
+    "react-dom": "^16.8.6",
     "react-native": "0.59.9",
     "react-native-gesture-handler": "^1.3.0",
+    "react-native-google-maps-directions": "^2.1.0",
```

Comment:
You do not need this package. Please remove it.

Reply:
Roger that. I found and started using the more versatile package for the directions. 
Thumbs up if we replace the previous package with react-native-navigation-directions?

Reply:
the downside for this is that when it opens up in the default iOS map app, there's no biking option. Only walking, transit, driving and something that looks like a hitchhiker. The google-maps-directions worked for showing the bike route. If the user didn't have google maps, it would just open up in their web browser.